### PR TITLE
Added WT20-H1 (HIBI JUNIOR)

### PR DESCRIPTION
**Pull request by wilba**
## Description

Added WT20-H1 (HIBI JUNIOR)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

File: keyboards/wilba_tech/wt20_h1/info.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                {"matrix": [3, 2], "x": 2, "y": 3},
+                {"matrix": [3, 3], "x": 3, "y": 3},
+                {"matrix": [4, 0], "x": 0, "y": 4},
+                {"matrix": [4, 1], "x": 1, "y": 4},
+                {"matrix": [4, 2], "x": 2, "y": 4},
+                {"matrix": [4, 3], "x": 3, "y": 4}
+            ]
+        }
+    }
+}
```
Comment (dunk2k) [inline keyboards/wilba_tech/wt20_h1/info.json:74]:
```suggestion
}

```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/wilba_tech/wt20_h1/info.json (hunk 1 above)
Comment (wilba) [reply keyboards/wilba_tech/wt20_h1/info.json:74]: this is the output of `qmk.exe format-json -i`

File: keyboards/wilba_tech/wt20_h1/config.h
Comment (drashna) [inline keyboards/wilba_tech/wt20_h1/config.h:1]: This file can be removed.

File: keyboards/wilba_tech/wt20_h1/info.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["F4", "F0", "B2", "B1"],
+        "rows": ["B3", "F1", "F5", "F6", "F7"]
+    },
+    "processor": "atmega32u4",
```
Comment (drashna) [inline keyboards/wilba_tech/wt20_h1/info.json:20]:
```suggestion
    "processor": "atmega32u4",
    "qmk": {
        "locking": {
            "enabled": true,
            "resync": true
        }
    },
```

File: keyboards/wilba_tech/wt20_h1/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_ortho_5x4(
+    KC_NUM,  KC_PSLS, KC_PAST, KC_PMNS,
+	KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+	KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
+	KC_P1,   KC_P2,   KC_P3,   KC_PENT,
+	KC_P0,   KC_P0,   KC_PDOT, KC_PENT
```
Comment (drashna) [inline keyboards/wilba_tech/wt20_h1/keymaps/default/keymap.c:12]:
```suggestion
    KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
    KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
    KC_P1,   KC_P2,   KC_P3,   KC_PENT,
    KC_P0,   KC_P0,   KC_PDOT, KC_PENT
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**